diff --git a/node_modules/expo-notifications/ios/ExpoNotifications/Notifications/DateComponentsSerializer.swift b/node_modules/expo-notifications/ios/ExpoNotifications/Notifications/DateComponentsSerializer.swift
index 3d70480..31f3b6c 100644
--- a/node_modules/expo-notifications/ios/ExpoNotifications/Notifications/DateComponentsSerializer.swift
+++ b/node_modules/expo-notifications/ios/ExpoNotifications/Notifications/DateComponentsSerializer.swift
@@ -5,11 +5,6 @@ class DateComponentsSerializer {
     serializedComponents["calendar"] = dateComponents.calendar?.identifier ?? NSNull()
     serializedComponents["timeZone"] = dateComponents.timeZone?.identifier ?? NSNull()
     serializedComponents["isLeapMonth"] = dateComponents.isLeapMonth ?? false
-    if #available(iOS 26.0, *) {
-      serializedComponents["isRepeatedDay"] = dateComponents.isRepeatedDay ?? false
-    }
-
-
     let map = calendarUnitsConversionMap()
     for (calendarUnit, keyName) in map {
       if let unitValue = dateComponents.value(for: calendarUnit) {
